### PR TITLE
Updated Congrats.story.jsx

### PR DESCRIPTION
**Pull request by mgc1194**
This PR updates story books for Congrats React files.

### CODE REVIEW

**Review by fisher-alice — COMMENTED** [review]
Thanks for doing this, Mario! 

When viewing the stories, I saw that for the Congrats component, there seems to be quite a few duplicate stories since the certificates are not rendered. Perhaps you could decrease the number of stories so that there are visible differences among the stories such as the difference between 'Congrats Signed In Student' vs 'Congrats Signed Out Student'.

When we talked offline, you explained that I needed to scroll down to view the entire component. You could include a couple screenshots of the stories in your PR description and state the 3 files you refactored. 

Nice job overall!

File: apps/src/templates/certificates/StudentsBeyondHoc.story.js [hunk 1]
```diff
@@ -1,203 +1,128 @@
...
+  isEnglish: true
+};
+
+export const ApplabSignedOutEnglish = Template.bind({});
+ApplabSignedOutEnglish.args = {
+  ...ApplabSignedInEnglish.args,
+  userType: 'signedOut'
+};
+
+export const pre2017MinecraftSignedInEnglishUnder13 = Template.bind({});
```
Comment (nicklathe) [inline apps/src/templates/certificates/StudentsBeyondHoc.story.js]: This exported function and the following 2 are lowercase, whereas the others are uppercase. Should we standardize on uppercase exported functions?

Comment (mgc1194) [reply apps/src/templates/certificates/StudentsBeyondHoc.story.js]: I just kept the original test name as much as I was able to, but I agree with your point. I am going to change this.

**Review by nicklathe — APPROVED** [review]
Looks good :shipit:

**Pull request closed without merging**